libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405WING-bd/hwdef-bl.dat

```diff
@@ -0,0 +1,4 @@
+# hw definition file for processing by chibios_pins.py
+# for SpeedyBeeF405WING bootloader
+
+include ../SpeedyBeeF405WING/hwdef-bl.dat
```

Comment:
Remove this file and the compiled bootloader (you've done well in this PR if you were creating an entirely new board, BTW - but you're not :-)